qmt/materials/mat_data.py

```diff
@@ -0,0 +1,62 @@
+from ..geometry.geo_data_base import GeoData
+from qmt.materials import Materials
+from qmt.geometry import Geo3DData, Geo2DData
+from typing import Dict
+from qmt.infrastructure import WithParts
+from dataclasses import dataclass
+
+
+@dataclass
+class MatPart:
+    """Object containing material information for a part.
+    Currently only contains one string. Might contain more things in the future.
```

Comment:
Do you have an example as to what this would be in the future?

Reply:
No I can't think of any right now, but I thought it was good for encapsulation in case that need arises in the future. It is also to parallel GeoData containing GeoParts, TFData containing TFParts, etc